config/styleguide/config.js

```diff
@@ -48,6 +48,7 @@ module.exports = {
     '**/*.spec.tsx',
     '**/components/Fabric/*',
     '**/components/Icon/icons/*',
+    '**/components/Illustration/sizedIllustrations/*',
```

Comment:
looks like this path needs to be updated to:
`'**/components/Illustration/illustrations/*',`